process/interface.go

```diff
@@ -195,7 +195,7 @@ type IntermediateProcessorContainer interface {
 	Keys() []block.Type
 }
 
-// PreProcessorsContainerFactory defines the functionality to create an PreProcessors container
+// IntermediateProcessorsContainerFactory defines the functionality to create an PreProcessors container
```

Comment:
The name is updated but the comment is still related to `PreProcessors`

Reply:
done